launcher/InstanceCopyTask.cpp

```diff
@@ -134,6 +144,24 @@ void InstanceCopyTask::copyFinished()
     }
     if (m_useLinks)
         inst->addLinkedInstanceId(m_origInstance->id());
+    if (m_useLinks) {
+        auto allowed_symlinks_file = QFileInfo(FS::PathCombine(inst->gameRoot(), "allowed_symlinks.txt"));
+
+        QByteArray allowed_symlinks;
+        if (allowed_symlinks_file.exists()) {
+            allowed_symlinks.append(FS::read(allowed_symlinks_file.path()));
+            if (allowed_symlinks.right(1) != "\n")
+                allowed_symlinks.append("\n");  // we want to be on a new line
+        }
+        allowed_symlinks.append(m_origInstance->gameRoot().toUtf8());
+        allowed_symlinks.append("\n");
+        if (allowed_symlinks_file.isSymLink())
+            FS::deletePath(allowed_symlinks_file
+                               .path());  // we dont want to modify the origonal. also make sure the resulting file is not itself a link.
```

Comment:
typo here (also in the commit message ig)

```suggestion
                               .path());  // we dont want to modify the original. also make sure the resulting file is not itself a link.
```